docs2/installation.rst

```diff
@@ -56,7 +56,22 @@ First Installation
     .. note::
        symlink-install allows you to edit Python and config files without running `colcon build` every time.
 
-5. Set up software-in-the-loop simulation (optional)
+5. Set up Crazyradio
+
+   For the crazyradio, you need to setup usb rules in order to communicate with the Crazyflie. Find the instructions for that here `in Bitcraze's USB permission guide for Linux <https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/installation/usb_permissions/>`_.
```

Comment:
we also have a pc_permissions.sh file in this repo. Perhaps we should just delete it, since there will be changes with Crazyradio2 anyways and we want to avoid duplicating instructions?

Reply:
yes I'm all in for not have duplicate instructions. I'll remove the .sh file and merge